sdk/core/azure-core-amqp/inc/azure/core/amqp/internal/amqp_settle_mode.hpp

```diff
@@ -14,13 +14,13 @@ namespace Azure { namespace Core { namespace Amqp { namespace _internal {
     Mixed,
   };
 
-  std::ostream& operator<<(std::ostream& os, SenderSettleMode const& mode);
+  std::ostream& operator<<(std::ostream& os, SenderSettleMode const mode);
```

Comment:
A quick comment: Rather than wasting your time on multiple suggestions, it's probably faster and easier to just leave one comment and ask me to fix all of the similar occurrences. I can do it much more thoroughly using find and replace than you can with visual inspections.

Reply:
Sorry, I was trying to be more helpful, but I see how spamming these comments could be a problem as well. But it is actually better for both of us to only comment once per category of issues. I'll use this approach for the future comments.